### PR TITLE
Use feeds for printing instructions and syscalls

### PR DESCRIPTION
**Pull request by kvalakuzhyzp**

### CODE REVIEW

File: tests/test_trace.py [hunk 1]
```diff
@@ -33,39 +33,48 @@
...
             self.assertNotIn("[080f5c00]", stdout.getvalue())
             self.assertIn("[08131b16]", stdout.getvalue())
 
-    def test_traceoff(self):
-        z = Zelos(path.join(DATA_DIR, "static_elf_helloworld"), verbosity=1)
-        z.plugins.trace.traceoff(0x08048B73)
+    def test_trace_inst_stop_on_addr(self):
+        z = Zelos(
+            path.join(DATA_DIR, "static_elf_helloworld"),
+            inst_feed="",
```
Comment (rcourt) [inline tests/test_trace.py:50]: You can accomplish the same thing with `inst_feed=True`, which may be clearer for someone who is unfamiliar with the details of ZML but wants to activate the instruction feed.

Comment (rcourt) [reply tests/test_trace.py:50]: On the other hand, if changed, may confuse someone who is unfamiliar into thinking that this arg must be a boolean.

File: tests/test_linux_arm.py [hunk 2]
```diff
@@ -26,12 +26,12 @@
 
 
 class ZelosTest(unittest.TestCase):
-    def test_static_elf_verbose(self):
+    def test_static_elf_code_feed(self):
         z = Zelos(
             path.join(DATA_DIR, "static_elf_arm_helloworld"),
-            tracethread="none",
+            shutup=True,
+            code_feed="",
```
Comment (rcourt) [inline tests/test_linux_arm.py:33]: code_feed?

File: tests/test_trace.py (hunk 1 above)
Comment (kvalakuzhyzp) [reply tests/test_trace.py:50]: This is a good point. I think the presence of flags should be indicated by a boolean variable for keyword arguments. Changing test usages to this.

File: tests/test_linux_arm.py (hunk 2 above)
Comment (kvalakuzhyzp) [reply tests/test_linux_arm.py:33]: Annnnnddd yeah you're right. It's an inst_feed

**Pull request merged**